invenio_records_marc21/translations/de/LC_MESSAGES/messages.po

```diff
@@ -8,77 +8,109 @@ msgid ""
 msgstr ""
 "Project-Id-Version: invenio-records-marc21 0.2.1\n"
 "Report-Msgid-Bugs-To: info@tugraz.at\n"
-"POT-Creation-Date: 2021-11-09 12:13+0100\n"
+"POT-Creation-Date: 2023-03-13 07:48+0100\n"
 "PO-Revision-Date: 2021-11-09 12:13+0100\n"
 "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
 "Language: de\n"
 "Language-Team: de <LL@li.org>\n"
-"Plural-Forms: nplurals=2; plural=(n != 1)\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Generated-By: Babel 2.9.1\n"
+"Generated-By: Babel 2.11.0\n"
 
-#: invenio_records_marc21/errors.py:40
+#: invenio_records_marc21/config.py:32
+msgid "Best match"
+msgstr "Höchste Übereinstimmung"
+
+#: invenio_records_marc21/config.py:36
+msgid "Newest"
+msgstr "Neuester"
+
+#: invenio_records_marc21/config.py:118 invenio_records_marc21/config.py:125
+#: invenio_records_marc21/config.py:140 invenio_records_marc21/config.py:148
+#: tests/conftest.py:141 tests/conftest.py:148
+msgid "DOI"
+msgstr "DOI"
+
+#: invenio_records_marc21/errors.py:46
 msgid ""
 "The table you are looking for does not exist.\n"
 "Maybe you missed to re-setup the services after the marc21 module was "
 "installed.\n"
 "Try to do a invenio-cli services setup -f"
 msgstr ""
 
-#: invenio_records_marc21/errors.py:45
+#: invenio_records_marc21/errors.py:51
 msgid ""
 "Connection to Services refused.\n"
 "Maybe you missed to start the services.\n"
 "Try to do a invenio-cli services setup -f"
 msgstr ""
 
-#: invenio_records_marc21/services/config.py:42
+#: invenio_records_marc21/errors.py:56
+msgid "User has not the permissions."
+msgstr "Benutzer hat nicht die erforlderichen Berechtigung."
+
+#: invenio_records_marc21/services/config.py:50
+#: invenio_records_marc21/templates/invenio_records_marc21/record.html:46
 msgid "Access status"
-msgstr ""
+msgstr "Zugriffsstatus"
 
-#: invenio_records_marc21/services/config.py:44
+#: invenio_records_marc21/services/config.py:52
 msgid "Public"
-msgstr ""
+msgstr "Öffentlich"
 
-#: invenio_records_marc21/services/config.py:45
+#: invenio_records_marc21/services/config.py:53
 msgid "Embargoed"
-msgstr ""
+msgstr "Gesperrt"
 
-#: invenio_records_marc21/services/config.py:46
+#: invenio_records_marc21/services/config.py:54
 msgid "Restricted"
-msgstr ""
+msgstr "Eingeschränkt"
 
-#: invenio_records_marc21/services/config.py:52
+#: invenio_records_marc21/services/config.py:60
 msgid "State"
-msgstr ""
+msgstr "Status"
 
-#: invenio_records_marc21/services/config.py:53
+#: invenio_records_marc21/services/config.py:61
 msgid "Published"
-msgstr ""
+msgstr "Veröffentlicht"
 
-#: invenio_records_marc21/services/config.py:53
+#: invenio_records_marc21/services/config.py:61
 msgid "Unpublished"
-msgstr ""
+msgstr "Unveröffentlicht"
+
+#: invenio_records_marc21/services/schemas/__init__.py:34
+msgid "Invalid persistent identifier scheme."
+msgstr "Ungültiges Schema für permanente Kennung."
```

Comment:
This german translation seams not right?

Reply:
Same here